common/services/frameworks.js

```diff
@@ -100,6 +100,8 @@ const frameworksService = {
       questions,
       type,
       validations = [],
+      minItems = 1,
```

Comment:
We'll need to actually define these in the framework before we merge it. Slightly pedantic but the casing style from the framework has been snake case so we may need to update this once we get the definition into the framework.

Reply:
@cwrw we should have a quick chat about how/if we actually want to define this in the framework at this stage.